docs/workflow/building/libraries/freebsd-instructions.md

```diff
@@ -66,7 +66,7 @@ index 768b6fb..c02f76f 100644
 +++ b/repos/core-setup.proj
 @@ -4,6 +4,7 @@
    <PropertyGroup>
-     <BuildArguments>-ConfigurationGroup=$(Configuration) -PortableBuild=$(PortableBuild) -SkipTests=true </BuildArguments>
+     <BuildArguments>-Configuration=$(Configuration) -PortableBuild=$(PortableBuild) -SkipTests=true </BuildArguments>
```

Comment:
You are changing something in core-setup here. Even though this documented is completely outdated I would revert the change.